### PR TITLE
[devtools-translate] Korean - What's New in DevTools (Chrome 106)

### PR DESCRIPTION
**Pull request by cwdoh**
Due to #3738

Changes proposed in this pull request:

- translated what's new in devtools 106 into Korean

### CODE REVIEW

**Comment by jecfish** [comment]
@cwdoh please sign the CLA

**Comment by jecfish** [comment]
Thanks Changwook! @TORU0239 @techhtml please help to review!

File: site/ko/blog/new-in-devtools-106/index.md [hunk 1]
```diff
@@ -28,26 +28,23 @@ draft: true
...
 <!-- + 5. Update the whats-new.md file -->
 
-<!-- ## Group files by Authored / Deployed in the Sources panel {: #authored } -->
-
-<!-- The **Group files by Authored / Deployed** is now shown under the 3-dot menu. Previously, it showed directly on the navigation pane. -->
-
-<!-- Open this [demo](https://ng-devtools.netlify.app/). Enable the **Group files by Authored / Deployed** setting to view your original source code (Authored) first and navigate to them quicker. -->
+## 소스 패널에서 작성/배포 단위로 파일 그룹화하기
 
+**작성/배포 단위로 파일 그룹화하기**는 이제 3점 메뉴에 위치합니다. 이전에는 네비게이션 패널에서 바로 확인할 수 있었습니다.
```
Comment (techhtml) [inline site/ko/blog/new-in-devtools-106/index.md:33]: 뭔가 Three-dot 이 3점 메뉴라는 게 어색한 거 같은데, 마땅히 개선할만한 번역은 없어보이네요.

**Review by techhtml — COMMENTED** [review]
Approved :)

File: site/ko/blog/new-in-devtools-106/index.md (hunk 1 above)
Comment (cwdoh) [reply site/ko/blog/new-in-devtools-106/index.md:33]: `Kebab Menu Icon` is its known name, but I can't believe KR people recognize it. So, I took a some familiar name for it. :/

File: site/ko/blog/new-in-devtools-106/index.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ tags:
 draft: true
```
Comment (jecfish) [inline site/ko/blog/new-in-devtools-106/index.md:15]:
```suggestion
```

**Review by jecfish — APPROVED** [review]

**Comment by jecfish** [comment]
Thanks all!

**Pull request merged**